components/scream/src/share/field/field_monotonicity_check.hpp

```diff
@@ -39,7 +39,7 @@ class FieldMonotonicityCheck: public FieldPropertyCheck<RealType> {
     return false;
   }
 
-  void repair(Field<RealType>& field) const override {
+  void repair(Field<RealType>& /* field */) const override {
```

Comment:
Just curious: did a compiler generate a warning for this unused parameter? I find these warnings to be distracting and usually disable them myself. Omitting unused parameters is Yet Another Thing that people must remember when using C++. :-)

Reply:
In my vim editor, clang compiles the code in the background, and highlights warnings/errors. I think it uses pedantic flags. It really helps reduce the edit->compile cycles. This was one of them.

I agree that these warnings are not that important. Alas, I like to see my build output consisting just of 
```
[ 67%] Building CXX object ...
[ 68%] Built target ...
```
without any warning to pollute the output.